### PR TITLE
Remove manpages and other documentation, cleanup empty doc dirs.

### PR DESCRIPTION
**Pull request by sil2100**
Quick trivial addition (if, of course, we don't want manpages).

### CODE REVIEW

File: hooks/900-cleanup-etc-usr-var.chroot [hunk 1]
```diff
@@ -47,6 +47,13 @@ rm -f /etc/mke2fs.conf
 # python deb install leftovers
 rm -rf /usr/local/lib/python*
 
+# we don't want any manpages or documentation
+rm -rf /usr/share/man /usr/share/groff /usr/share/info \
+       /usr/share/lintian /usr/share/linda /var/cache/man
+
+# clean up unneeded empty directories in docs
+find binary/boot/filesystem.dir/usr/share/doc -empty  -print0 | xargs -0 rmdir || true
```
Comment (mvo5) [inline hooks/900-cleanup-etc-usr-var.chroot]: This path looks not quite right, probably a copy/paste issue.

File: hooks/900-cleanup-etc-usr-var.chroot [hunk 2]
```diff
@@ -47,6 +47,13 @@ rm -f /etc/mke2fs.conf
 # python deb install leftovers
 rm -rf /usr/local/lib/python*
 
+# we don't want any manpages or documentation
+rm -rf /usr/share/man /usr/share/groff /usr/share/info \
```
Comment (mvo5) [inline hooks/900-cleanup-etc-usr-var.chroot:51]: Its interesting that most of these are pretty empty already (and some like linda do not exit on core18). I wonder if the base tarball we download already cleans the man-pages?

Comment (sil2100) [reply hooks/900-cleanup-etc-usr-var.chroot:51]: Probably, might be something to check in free time. Should I remove those that are not existent from the list? It's a copy-paste from core and not sure if there's anything wrong in having those here just-in-case.

**Comment by vorlonofportland** [comment]
I see this PR has been open for a while; @sil2100 do you still think it's relevant?

A lot of these changes seem to duplicate changes that are already part of the "minimized" profile in livecd-rootfs.  I think we should ensure that the base tarball we're using as a source is minimized, and if there are other things that should be removed, we should evaluate landing those changes in livecd-rootfs first.

**Pull request closed without merging**